augur/filter/subsample.py

```diff
@@ -1,18 +1,26 @@
-from collections import defaultdict
 import heapq
 import itertools
 import uuid
-import numpy as np
-import pandas as pd
+from collections import defaultdict
 from textwrap import dedent
 from typing import Collection, Dict, Iterable, List, Optional, Set, Tuple, Union
 
-from augur.dates import get_year_week, get_year_month_day
+import numpy as np
+import pandas as pd
+
+from augur.dates import get_year_month_day, get_year_week
 from augur.errors import AugurError
 from augur.io.metadata import METADATA_DATE_COLUMN
-from augur.io.print import print_err, _n
+from augur.io.print import _n, print_err
+
 from . import constants
-from .weights_file import WEIGHTS_COLUMN, COLUMN_VALUE_FOR_DEFAULT_WEIGHT, get_default_weight, get_weighted_columns, read_weights_file
+from .weights_file import (
+    COLUMN_VALUE_FOR_DEFAULT_WEIGHT,
+    WEIGHTS_COLUMN,
+    get_default_weight,
+    get_weighted_columns,
+    read_weights_file,
+)
```

Comment:
I'm fine with the formatting changes in 8034fea78f160e30b88bd43a16cb4357e9035edf, but can we have a better commit message than "f"?

Reply:
I'll squash my commits! I'm undoing the sort as well, let's decide separately if we want to do that or not. I should just disable isort on Augur